draft.md

```diff
@@ -241,62 +245,69 @@ Emotiq is private, scalable, and natural. Research-backed and supported by a tea
-
-**Kieran Costello**
-Chief Wordsmith
-
-Kieran has written for the market’s leading tech companies and start-ups, and has been a core contributor to numerous ground-breaking blockchain projects. As a writer by habit and trade, he specializes in creating stories and brand identities for ‘edge companies’ – those innovating in uncharted spaces and offering new solutions to legacy problems. This naturally led him to focus on blockchain-based solutions, and specifically to Emotiq.
-
-**Emma Cooper**
-Global Community Manager
-
-Emma is the Community Manager for Emotiq.  Prior to joining the team, Emma worked for many years in front-line counterterrorism for the US Department of Homeland Security, before which she was a lawyer in the US, focusing primarily on private placements and corporate law.
+Ever since cobbling together a distributed system to process data reduction on unused nighttime nodes of the Cornell astronomy network in the late 1980s, Mark Evenson has been involved in plumbing the depths of what is possible to compute across the Internet. From modeling multi-party business process execution orchestration in description logic to hot-patching the hosting infrastructure for surging web traffic across the Western hemisphere, his work has repeatedly combined a future sketched by academic wizardry with the needs of the pragmatic now. He is excited to solve the problems inherent in developing and deploying a decentralized, planetary-scale sharded ledger.
 
 **Anna Movchaniuk**
 PR Maven
 
-Anna is a proactive, dedicated, and detail-oriented corporate communications practitioner with extensive public relations experience in Europe and Canada.  She has worked on projects in industries as diverse as finance, sporting events, and film.
+Anna is a proactive, dedicated, and detail-oriented corporate communications practitioner with extensive public relations experience in Europe and Canada. She has worked on projects in industries as diverse as finance, sporting events, and film.
 
 **Wen Qiang**
 China Community Manager
 
 Wen is the China Community Manager for Emotiq. With a deep knowledge of the Chinese cryptocurrency space and thorough understanding of blockchain projects, he provides the Emotiq team with expert regional advice, in addition to offering Chinese language support across all of Emotiq's social media channels.
 
+**Eric Yao**
+Writer and Editor
+
+Eric is the writer and editor for Emotiq. He has deep experience researching and writing about the cryptocurrency space and blockchain technology. He is excited to help create clear and effective PR, content, copy, and communications.
+
+**Ann Söderblom** Project Manager
```

Comment:
Let’s make it “Marketing” instead of “Project Manager”.

Reply:
Done!